### ISSUE TITLE
telemetryDashboard-1-Generate-Metadata task taking time to reload

### ISSUE THREAD

**Issue by Roshan0312** [eapowertools/qs-telemetry-dashboard#78]
Hi Team,

I have installed the Telemetry Dashboard but while run this telemetryDashboard-1-Generate-Metadata task it is taking more time  to reload. 

As i observed it  is stuck in started status. Can you please help me to resolve this issue.
![Uploading Telemetry.PNG…]()

**Comment by jparis** [comment]
How many apps do you have? How big are they? How long has it been running for?

**Comment by levi-turner** [comment]
Is this still an issue?

**Comment by armandfrigo** [comment]
Hi Jesse and Levi, we face a similar issue with Qlik Sense which host a large number of applications.
Typically, one of our development dedicated Qlik Sense site hosts 23 500 applications. The telemetryDashboard-1-Generate-Metadata task does not manage to complete its execution within 24 hours, then fails.
Would it be possible to introduce a flag to filter the apps to be processed? like Published/Unpublished or based on the lastmodifieddate? Or anything smarter you can think of? I don't think there is any sort of incremental update right now, but we probably don't need to extract the metadata from apps which have not been touched.

**Comment by jparis** [comment]
Yeah I realize that, need to chat internally about what a solution would be here.
